server/accounts.go

```diff
@@ -4215,6 +4273,13 @@ func (tr *transform) transformSubject(subject string) (string, error) {
 	return tr.transform(tts)
 }
 
+func (tr *transform) getHashPartition(key string, numBuckets int) string {
+	h := fnv.New32a()
+	h.Write([]byte(key))
+
+	return fmt.Sprintf("%d", h.Sum32()%uint32(numBuckets))
```

Comment:
I'd imagine, not having to read the format string helps.
`strconv.Itoa(h.Sum32()%uint32(numBuckets))`

Reply:
You are correct that this being one of the few places actually in the code path for every message being published on a matching subject every little optimization does add up changed to `strconv.Itoa(h.Sum32()%uint32(numBuckets))`